test/kitchen/test/integration/dd-agent-upgrade-agent7/rspec/dd-agent-upgrade-agent7_spec.rb

```diff
@@ -0,0 +1 @@
+../../dd-agent-upgrade/rspec/dd-agent-upgrade_spec.rb
```

Comment:
Is this the expected contents of these files??

Reply:
Yes, this is how Github represents symbolic links 😅. If you click on "View file", you'll see this: https://github.com/DataDog/datadog-agent/blob/bc01b0923ee5ba3513e5853564623c9c90ca4dc2/test/kitchen/test/integration/dd-agent-upgrade-agent7/rspec/dd-agent-upgrade-agent7_spec.rb which more clearly states so.